<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UClassesListWidget</class>
 <widget class="QWidget" name="UClassesListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tabStorageByName">
      <attribute name="title">
       <string>By Name</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QListWidget" name="listWidgetStorageByName"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabStorageByLibs">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <attribute name="title">
       <string>By Libs</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item>
        <widget class="QTreeWidget" name="treeWidgetStorageByLibs">
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabLibsCtrl">
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <attribute name="title">
       <string>Libs Control</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="lineEditSearch"/>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxLibList">
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>Library List</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QListWidget" name="listWidgetRTlibs">
            <property name="focusPolicy">
             <enum>Qt::ClickFocus</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxLibComps">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>Library Components</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QListWidget" name="listWidgetRTlibClasses"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="ButtonsPanel">
         <property name="spacing">
          <number>3</number>
         </property>
         <item>
          <widget class="QPushButton" name="LoadRTlibButton">
           <property name="text">
            <string>Load Library</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="CreateRTlibButton">
           <property name="text">
            <string>Create Runtime Library</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="DeleteLibButton">
           <property name="text">
            <string>Delete Library</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="RenameRTlibButton">
           <property name="text">
            <string>Rename Runtime Library</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="AddNewClassButton">
           <property name="text">
            <string>Add New Class</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="DeleteClassButton">
           <property name="text">
            <string>Delete Class</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
